Celeste.Mod.mm/Mod/Everest/Everest.Content.cs

```diff
@@ -801,6 +803,10 @@ public static void Add(string path, ModAsset metadata)
                     // Special case: Fallback type for anything inside of the Ahorn folder.
                     // Will be ignored during collision checks.
                     type = typeof(AssetTypeAhorn);
+                } else if (file.StartsWith("Loenn/")) {
```

Comment:
the Loenn directory can also be `Lönn`, so would add that check as well

Reply:
re: cruor this is no longer used, and couldn't find mods using it in the database so not needed